custom_components/unifiprotect/light.py

```diff
@@ -71,56 +72,64 @@ def hass_to_unifi_brightness(value):
 class UnifiProtectLight(UnifiProtectEntity, LightEntity):
     """A Ubiquiti Unifi Protect Light Entity."""
 
-    def __init__(self, upv_object, protect_data, server_info, light_id):
+    def __init__(self, protect, protect_data, device):
         """Initialize an Unifi light."""
-        super().__init__(upv_object, protect_data, server_info, light_id, None)
-        self._name = self._device_data["name"]
+        super().__init__(protect, protect_data, device, None)
+        self.device: Light = device
+        self._attr_name = self.device.name
         self._attr_icon = "mdi:spotlight-beam"
         self._attr_supported_features = SUPPORT_BRIGHTNESS
-        self._attr_is_on = self._device_data["is_on"] == ON_STATE
+        self._async_update_device_from_protect()
```

Comment:
TODO: check this to make sure it doesn't write the state twice at startup

Reply:
It is not. That is why I made `_async_update_device_from_protect` (as opposed to the existing `_async_updated_event`)